src/sentry/tasks/integrations/github/open_pr_comment.py

```diff
@@ -305,60 +305,93 @@ def get_top_5_issues_by_count_for_file(
 
     multi_if = language_parser.generate_multi_if(function_names)
 
+    # fetch the count of events for each group_id
+    subquery = (
```

Comment:
do you have to do this in Snuba, or do you think we could get through this through the postgres models?

Reply:
all of the information together (title, culprit, function name, stacktrace frame) is pullable from snuba. this raw data is how we populate the postgres models